aspnetcore/fundamentals/logging/index/samples/6.x/WebLog/Program.cs

```diff
@@ -1,58 +1,80 @@
-#define APP3 // FIRST SECOND APP3
+#define FIRST // FIRST SECOND APP3 APP4
 #if NEVER
 #elif FIRST
-#region snippet1
+// <snippet1>
 var builder = WebApplication.CreateBuilder(args);
 
-var logger = LoggerFactory.Create(config =>
-{
-    config.AddConsole();
-}).CreateLogger("Program");
+// Configure logging with AddConsole()
+builder.Logging.AddConsole();
 
 var app = builder.Build();
 
 app.MapGet("/", () => "Hello World!");
 
-app.MapGet("/Test", async context =>
+app.MapGet("/Test", async (ILogger logger, HttpResponse response) =>
```

Comment:
```suggestion
app.MapGet("/Test", async (ILogger<Program> logger, HttpResponse response) =>
```
As-is it will build, but fail when run. There is no ILogger interface in the DI container, only ILogger\<T>.